### ISSUE TITLE
Jar file of the latest 1.13.2

### ISSUE THREAD

**Issue by TomKell** [SquidDev-CC/CC-Tweaked#146]
Hey!
I had the problem with the mismatched mod channel list.
You told that you fixed this problem in the latest commit..
Now my question.. Can you send me a link to this new version as a .jar?
I cant compile the git repo..

Hope you can help me!
King regards!!!

**Comment by miathedev** [comment]
"King regards"?

Give me some minutes. I will upload it here soon. Im building you the current master branch.

**Comment by TomKell** [comment]
Haha yeah my fault i wanted to write kind regards :P
Thank you!!!!!

**Comment by miathedev** [comment]
Maybe its interesting for you. I mirrored this repo to gitlab to be able to use the gitlab shared runners.
You can see the progress of the build here:
https://gitlab.com/miathedev/CC-Tweaked/-/jobs/179063699

**Comment by miathedev** [comment]
@SquidDev i could manage to create a nightly build jenkins server. Is there any use for sth like that?

**Comment by miathedev** [comment]
The build failed. My mistake.

**Comment by SquidDev** [comment]
Right, sorry for the delay - I've been aware from the computer all morning. I'm hoping to get an "actual proper release" out sometime next week - there's a couple of other bugs I need want to fix first. For the time being [this file](https://www.dropbox.com/s/ho7gqx7xs16hu6o/cc-tweaked-1.13.2-1.81.1-network-fix.jar?dl=1) should work.

> I could manage to create a nightly build jenkins server. Is there any use for that?

I could probably set Travis up to publish to my Maven or something - I'll have a fiddle.

**Comment by TomKell** [comment]
Thank you for the great support!
I will try it :)
Have a nice day!!!

**Comment by Wilma456** [comment]
@SquidDev It would be nice, if we have something like https://cc.crzd.me/ for CC-Tweaked

**Comment by TomKell** [comment]
![Screenshot_2](https://user-images.githubusercontent.com/7360469/54493807-65d16880-48d4-11e9-8e29-1de3ee34027d.png)

Still the same problem..

Mods:
![Screenshot_3](https://user-images.githubusercontent.com/7360469/54493829-90bbbc80-48d4-11e9-92a4-ff10d6c66b79.png)

on the latest Forge version

**Comment by SquidDev** [comment]
@TomKell Hrmr, I'm not able to reproduce on Forge 25.0.85 with just CC:T installed. Might be worth double checking you updated the client and server jar. If that still doesn't work, could you provide your logs for both?

Thanks!

**Comment by TomKell** [comment]
Yeah, I will try it all and tell you what happend!

**Comment by TomKell** [comment]
I set up a new server and a fresh client and only installed CC Tweaked.. and it works..
But when i install the other mods, I get the same error again..
Seems that CC dont like other mods :/

**Comment by TomKell** [comment]
Okayyyyy I will never know why but delete the ironshulkerboxes mod and it works...
But without the ironshulkerboxes mod, my world crashes.. so I just put the ironshulkerboxes mod back in the mod folder.. and everything works fine..

Thank you very much for this nice support and your patience!!!!
Love your work!

**Comment by SquidDev** [comment]
OK, so I _think_ everything is resolved on my end, so I'm gonna close this for now.

Let me know if you have any other issues!

**Comment by BigBang1112** [comment]
I'm still somehow reproducing this issue with only CC:T and like already 6 times reinstalled both client and server Forge. Tested other mods alone and those work fine.

Some points I found in the console:
- No data fixer registered for block entity [ComputerCraft entity name] (just a WARN though)
- Parsing error loading recipe [ComputerCraft entity name] - Invalid or unsupported recipe type
- Parsing error loading custom advancement [ComputerCraft entity name]: Disallowed data tag found

Not sure if it has any connections with this bug but yeah.
